.circleci/config.yml

```diff
@@ -150,49 +153,33 @@ commands:
           command: git submodule update --init
       - emsdk-env
       - npm-install
-      - run:
-          name: run tests
-          command: |
-            ./tests/runner << parameters.test_targets >>
-            $EMSDK_PYTHON ./tools/check_clean.py
-  run-tests-mac:
+      - pip-install
+  run-tests-base:
     description: "Runs emscripten tests"
     parameters:
       test_targets:
         description: "Test suites to run"
         type: string
     steps:
-      - attach_workspace:
-          # Must be absolute path or relative path from working_directory
-          at: ~/
-      - checkout
-      - emsdk-env
-      - npm-install
-      - run:
-          name: Install brew package dependencies
-          environment:
-            HOMEBREW_NO_AUTO_UPDATE: "1"
-          command: |
-            brew list cmake || brew install cmake
-            brew list python3 || brew install python3
       - run:
           name: run tests
           command: |
             ./tests/runner << parameters.test_targets >>
             $EMSDK_PYTHON ./tools/check_clean.py
+  run-tests:
```

Comment:
maybe "prep-and-run-tests"? Or maybe run-tests-base could be something else. I feel that the names are too close and it's not obvious what the difference is.